release.config.js

```diff
@@ -54,5 +71,8 @@ module.exports = {
       "prerelease": true
     }
   ],
-  plugins: branch === 'main' ? main_plugins : plugins,
+  plugins: branch === 'main' ? main_plugins : default_plugins,
 };
+
+
+console.log(module.exports);
```

Comment:
Did you want this console.log?

Reply:
Yeah it should just now print out in the Action which will make it easier to tell exactly what configuration options are being passed into semantic release.